src/index.ts

```diff
@@ -99,4 +99,13 @@ void (async () => {
 			return process.exit(ErrorCode.NotFoundRootPath);
 		}
 	}
+
+	if (args['serverUrl']) {
+		const serverUrl = args['serverUrl'];
+		try {
```

Comment:
Why try catch?

Reply:
Because i don't know for sure if i have to call a function or not so i kept it until i determine